### ISSUE TITLE
clojure-jack-in fails with Search failed: "(run-hooks 'slime-load-hook) ; on port"

### ISSUE THREAD

**Issue by locopati** [technomancy/swank-clojure#87]
In Emacs, using M-x clojure-jack-in with swank-clojure 1.4.0-SNAPSHOT causes
error in process filter: Search failed: "(run-hooks 'slime-load-hook) ; on port"

It does work to run lein swank + M-x slime-connect

**Comment by ngrunwald** [comment]
I have the same problem on emacs24 and 1.4.0-SNAPSHOT  .
It works fine on last stable version.

**Comment by gw666** [comment]
I, too, have the same problem with emacs24 and 1.4.0-SNAPSHOT. However, when I run `lein swank`, open my project's core.clj, and attempt M-x slime-connect, the minibuffer immediately responds with `(no matches)`. Any ideas on what's missing in my emacs such that `slime-connect` doesn't exist?

**Comment by ekoontz** [comment]
I hit this problem as soon as, on a whim, I did "lein upgrade". This upgraded my lein from version 1.4.2 to 1.6.2. I worked around this problem by downgrading back to lein 1.4.2 as follows:

``` cd
git clone https://github.com/technomancy/leiningen.git
cd leiningen
git checkout 1.4.2
mv ~/bin/lein /tmp
ln -s ~/leiningen/bin/lein ~/bin
```

Now `lein version` shows 1.4.2 and clojure-jack-in works again.

**Comment by technomancy** [comment]
This was caused by having an old version of swank. I've fixed it so that in clojure-mode 1.11.5 it should be fine with older versions now.

**Comment by ekoontz** [comment]
Hi technomancy, upgraded my clojure-mode to 1.11.5 (downloaded it from http://marmalade-repo.org/packages/clojure-mode-1.11.5.el , opened in emacs, and then `M-x package-install-from-buffer`).

Thereafter, I was able to `lein upgrade` again to the latest, and voila `M-x clojure-jack-in` works! Thanks. Hope this helps the other people who encountered this.
